common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -58,7 +39,23 @@
         % endfor
         <span id="answer_${id}"></span>
     </fieldset>
-
+    <div class="indicator_container">
+        % if input_type == 'checkbox' or not value:
+            <span class="status ${status.classname if show_correctness != 'never' else 'unanswered'}"
```

Comment:
Can we get a more semantic or meaningful classname here? `status` feels very general.